src/Integration.Vsix.UnitTests/CFamily/CFamily_CallAnalyzerTests.cs

```diff
@@ -40,7 +42,7 @@ public void CallAnalyzer_Succeeds_ReturnsMessages()
             var dummyProcessRunner = new DummyProcessRunner(MockResponse());
 
             // Act
-            var response = GetResponse(dummyProcessRunner, new Request(), new TestLogger());
+            var response = GetResponse(dummyProcessRunner, CreateRequest(), new TestLogger());
```

Comment:
The "call analyzer" tests no longer use the `Request` class - they use the interface instead.